### ISSUE TITLE
Lucene Dependency

### ISSUE THREAD

**Issue by rschwab68** [dexter/dexter#14]
Dexter depends on Lucene Version 4.1.0.
But I am using 5.3.0 in my project and APIs are not compatible.
It is possible to upgrade to Lucene 5.3.0 or higher?

**Comment by diegoceccarelli** [comment]
Hello @rschwab68,
we are writing a new version of Dexter at the moment, using the latest stable version of lucene (5.4.0), we'll publish it in a few weeks.

**Comment by rschwab68** [comment]
Hello @diegoceccarelli ,
ok, fine. Do you also change LuceneHelper? As you know I want to use german language and I need the german Umlaute. LuceneHelper uses the Unicode cleaner to map to US-ASCII encoding. Umlaute get lost.

**Comment by Arminmsg** [comment]
@rschwab68 
Hi guys, I'm currently also trying to train it on the german model and I came across the same problem, how did you fix the problem with Umlauten? Do you just map the ü and so on to u and then when you get a response back reverse the mapping?

@diegoceccarelli First of all, thank you for your work, trained my first model kind of successfully. 
Are there any new information about the new version of Dexter?

**Comment by diegoceccarelli** [comment]
@arminmsg and @rschwab68 I've a new version that supports utf8, it has tagme implemented and I've testing it in an English model. But i still have to write the documentation and evaluate the quality. Happy to share the code if you want to play with it.

**Comment by Arminmsg** [comment]
@diegoceccarelli 
Would be great

**Comment by rschwab68** [comment]
@diegoceccarelli yes, I would like to play with it. you dont want to upload to github as a new branch?

@Arminmsg I was waiting for the code update by @diegoceccarelli

**Comment by diegoceccarelli** [comment]
@rschwab68  i shared it with you, it is a private fork of Dexter on my account. There is branch with the new project ( in the folder dexter-core3) it is private because it is still instable and not documented ;) I'll publish it when I'm done :)

**Comment by rschwab68** [comment]
@diegoceccarelli thank you for sharing the fork. I am looking forward that it will be published.
